generators/server/templates/src/main/java/package/service/dto/AdminUserDTO.java.ejs

```diff
@@ -0,0 +1,185 @@
+package <%=packageName%>.service.dto;
+
+import <%=packageName%>.config.Constants;
+
+import <%=packageName%>.domain.Authority;
+import <%=packageName%>.domain.User;
+import io.micronaut.core.annotation.Introspected;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+
+import javax.validation.constraints.*;
+import java.time.Instant;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * A DTO representing a user, with their authorities.
+ */
+@Introspected
+public class AdminUserDTO {
+
+    private <%= userPkType %> id;
+
+    @NotBlank
+    @Pattern(regexp = Constants.LOGIN_REGEX)
+    @Size(min = 1, max = 50)
+    private String login;
+
+    @Size(max = 50)
+    private String firstName;
+
+    @Size(max = 50)
+    private String lastName;
+
+    @Email
+    @Size(min = 5, max = 254)
+    private String email;
+
+    @Size(max = 256)
+    private String imageUrl;
+
+    private boolean activated = false;
+
+    @Size(min = 2, max = 6)
+    private String langKey;
+
+    private String createdBy;
+
+    private Instant createdDate;
+
+    private String lastModifiedBy;
+
+    private Instant lastModifiedDate;
+
+    private Set<String> authorities;
+
+    public AdminUserDTO() {
+        // Empty constructor needed for Jackson.
+    }
+
+    public AdminUserDTO(User user) {
+        this.id = user.getId();
+        this.login = user.getLogin();
+        this.firstName = user.getFirstName();
+        this.lastName = user.getLastName();
+        this.email = user.getEmail();
+        this.activated = user.getActivated();
+        this.imageUrl = user.getImageUrl();
+        this.langKey = user.getLangKey();
+        //this.createdBy = user.getCreatedBy();
```

Comment:
Can this be removed?

Reply:
Yes, it was just copy-pasted from the original UserDTO